storefront/lib/spree/storefront/testing_support/cart_utils.rb

```
@@ -0,0 +1,30 @@
+    module TestingSupport
+      module CartUtils
+        def add_to_cart(product, goto_checkout = false)
+          visit spree.product_path(product)
+
+          click_button Spree.t(:add_to_cart)
+          wait_for_turbo
+          expect(page).to have_link('Checkout')
+
+          def view_cart
+            visit spree.cart_path
+          end
```

RuboCop (1.75.5)

[warning] Method definitions must not be nested. Use `lambda` instead.

(Lint/NestedMethodDefinition)